_episodes/04-ome-public-example.md

```diff
@@ -185,4 +174,18 @@ provider supports automatic port collision detection and resolution.
 ~~~
 {: .output}
 
+## Rerun the playbook:
+A well written Ansible playbook is *idempotent*, this means you can run it multiple times and it will only make changes to the server if necessary:
```

Comment:
```suggestion
A well-written Ansible playbook is *idempotent*, this means you can run it multiple times and it will only make changes to the server if necessary:
```